### PR TITLE
task: dockerize and update node

### PR DESCRIPTION
**Pull request by theworkflow**
- Dockerize service using `node@6.10`
- Lint code

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+services:
+  simple-queue:
+    container_name: simple-queue
+    image: xervo/simple-queue:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file: .env
+    environment:
+      NODE_ENV: DEVELOPMENT
```
Comment (jackboberg) [inline docker-compose.yml]: why uppercased?

Comment (theworkflow) [inline docker-compose.yml]: didn't think about it when i was typing. i can do lowercase

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,3 @@
+QUEUE_USERNAME=username
+QUEUE_PASSWORD=password
+
```
Comment (jackboberg) [inline .env.example]: add `QUEUE_PORT`

**Review by jackboberg — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+FROM node:6.10
+
+COPY . /mnt/app
+WORKDIR /mnt/app
+
```
Comment (jackboberg) [inline Dockerfile]: add `EXPOSE 80`

**Review by jackboberg — APPROVED** [review]

**Review by bsatrom — APPROVED** [review]

**Review by jackboberg — APPROVED** [review]

**Pull request merged**